README.md

```diff
@@ -102,76 +99,17 @@ terraform apply
 ```
 
 #### Per-user Infrastructure
-* Next build the infrastructure for your user, in the `infrastructure` directory of the Shared Infrastructure project:
-```
-cd infrastructure
-
-# you'll need to init whenever you add new providers in terraform
-# if prompted to migrate all workspaces to S3 then respond with 'yes'
-terraform init -backend-config "bucket=<app_name>-terraform-state"
-# select your workspace
-terraform workspace new <your_workspace>
-terraform apply
-```
+* Next build the infrastructure for your user
+* There is a  `build.sh <app_name> <workspace_name>` script in the `securityanalytics-sharedinfrastructure` directory - run this and it will set up the infrastructure for you. If you are using Windows and Git Bash and it doesn't start that up, see the note above in pre-requisites.
```

Comment:
build isn't really the correct word is it? It doesn't just build, it deploys too. setup?

Reply:
yeah ok